src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java

```diff
@@ -92,6 +97,27 @@ public BranchApi branch(String ref) throws RestApiException {
         JsonElement branches = gerritRestClient.getRequest(request);
         return branchInfoParser.parseBranchInfos(branches);
     }
+    
+    @Override
+    public ListRefsRequest<TagInfo> tags() {
+        return new ListRefsRequest<TagInfo>() {
+            @Override
+            public List<TagInfo> get() throws RestApiException {
+                return ProjectApiRestClient.this.getTagInfos(this);
+            }
+        };
+    }
+    
+    @Override
+    public TagApi tag(String ref) throws RestApiException {
+        return new TagApiRestClient(gerritRestClient, tagInfoParser, this, ref);
+    }
+    
+    private List<TagInfo> getTagInfos(ListRefsRequest<TagInfo> lbr) throws RestApiException {
```

Comment:
I've renamed the variable and now the url for tags is build in separate method like branchesUrl.
However currently Gerrit doesn't allow any tags retrieval options in contrast to branches options (limit, skip, substring, regex) and so that argument may be extra.